### ISSUE TITLE
Extend the functionality 

### ISSUE THREAD

**Issue by surajkadapa** [surajkadapa/hamming_code_1#3]
Add 11 data bits and 4 parity bits that follow even parity with a similar scheme given in the readme, make sure the circuit generates only the parity bits.

**Comment by skandesh-kk** [comment]
assign this to me

**Comment by surajkadapa** [comment]
@skandesh-kk Yes. The time limit is 1 hour. If not, others can ask to be assigned.

**Comment by ProSai45** [comment]
Hey Can You Assign Me This Issue after @skandesh-kk

**Comment by ProSai8055** [comment]
Hey Can You Assign Me This Issue after @skandesh-kk

**Comment by euphoricair7** [comment]
Could You assign me this issue?

**Comment by 411sst** [comment]
can i get assigned if this is still open?

**Comment by surajkadapa** [comment]
@ProSai8055 Yes. The time limit is 1 hour. If not, others can ask to be assigned.

**Comment by surajkadapa** [comment]
@ProSai8055 any update?
